client/reader/sidebar/index.jsx

```diff
@@ -175,7 +175,8 @@ export const ReaderSidebar = createReactClass( {
 									{ this.props.translate( 'Manage' ) }
 								</a>
 							</li>
-							{ config.isEnabled( 'reader/conversations' ) && (
+							{ config.isEnabled( 'reader/conversations' ) &&
```

Comment:
what happens if a user goes directly to the url?

Reply:
same thing that happens now if you're on prod

Reply:
oh. right. crap.

Reply:
hmmmm. looks like we need a sysreq either way to route /read/conversations to prod and not to atlas. 

my gut says you'd get to see the convos tool? we could try to gate it on /read/teams coming back from the sidebar somehow but ... i'm not really that worried about it?